.github/workflows/build-and-publish.yml

```diff
@@ -114,8 +114,10 @@ jobs:
           name: workspace
 
       # Using dotnet nuget push to publish the specified package to NuGet.org.
-      - name: Publish ${{ needs.set_variables.outputs.package_name }}
+      - name: Publish ${{ needs.set_variables.outputs.PACKAGE_NAME }}
         # Getting the NUGET_API_KEY from the org secrets.
         env:
           API_KEY: ${{ secrets.NUGET_API_KEY }}
-        run: dotnet nuget push ${{ needs.set_variables.outputs.package_name }}/bin/Release/${{ needs.set_variables.outputs.package_name }}.${{ needs.set_variables.outputs.package_version }}.nupkg --api-key ${{ env.API_KEY }} --source https://api.nuget.org/v3/index.json
+        # run: dotnet nuget push ${{ needs.set_variables.outputs.PACKAGE_NAME }}/bin/Release/${{ needs.set_variables.outputs.PACKAGE_NAME }}.${{ needs.set_variables.outputs.PACKAGE_VERSION }}.nupkg --api-key ${{ env.API_KEY }} --source https://api.nuget.org/v3/index.json
+        run: |
+          echo ${{ needs.set_variables.outputs.PACKAGE_NAME }}/bin/Release/${{ needs.set_variables.outputs.PACKAGE_NAME }}.${{ needs.set_variables.outputs.PACKAGE_VERSION }}.nupkg
```

Comment:
Was this for debugging that didn't get put back?

Reply:
Yes, I just noticed that.